src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -3015,7 +3016,7 @@ class CreateFunctionStatementSegment(BaseSegment):
         Sequence("OR", "REPLACE", optional=True),
         Ref("TemporaryGrammar", optional=True),
         "FUNCTION",
-        Sequence("IF", "NOT", "EXISTS", optional=True),
+        Ref("IfNotExistsGrammar", optional=True),
```

Comment:
On line 3016, just above here, it looks like you could also replace
```
Sequence("OR", "REPLACE", optional=True),
```
with
```
Ref("OrReplaceGrammar", optional=True),
```

Reply:
Good spot - done!

Reply:
nice, feel free to merge once the ci finishes 👍